QUARKUS-1159.md

```diff
@@ -0,0 +1,37 @@
+# QUARKUS-1159 - Quarkus FIPS support over RHEL8
+
+JIRA link: https://issues.redhat.com/browse/QUARKUS-1159
+
+The goal is ensure Quarkus runs on a FIPS enabled RHEL 8 System
+
+## Scope of testing
+
+### Impact on test suites and testing automation
+- Test will be implemented in https://github.com/quarkus-qe/quarkus-test-suite (Openshift scenarios are out of scope) 
+  
+### Impact on resources
+
+FIPS coverage will have its own instances with FIPS enabled and will cover JDK 11 and 17. The impact is pretty big because we will run the current Quarkus test suite twice, in JVM and native mode. The best approach is implement this coverage with a Jenkins matrix job in order to 
+split the execution between several workers. This apporach will reduce the execution time but will require more raw resources in terms of memory
+and CPU. 
+
+- Baremetal resources: 16 large (8Gb RAM + 4CPUs) VM instances
+- Native resources: 16 xlarge (16Gb RAM + 8CPUs) VM instances
+
+The above estimation include the whole execution in JVM 11 and 17
+
+## Known issues
+
+- [quarkus-smallrye-jwt](https://issues.redhat.com/browse/QUARKUS-2066)
+- [Keycloak](https://issues.redhat.com/browse/QUARKUS-2071)
+- [DB2](https://github.com/IBM/Db2/issues/43)
+- [Infinispan](https://github.com/quarkusio/quarkus/issues/25136)
+
```

Comment:
https://issues.redhat.com/issues/?jql=project%20%3D%20QUARKUS%20AND%20text%20~%20fips gives me more issues than just the 2 JIRAs listed above